CONTRIBUTING.md

```diff
@@ -119,26 +120,26 @@ The unit tests just check if the code runs and contains the required resource
 definitions. In those tests, the code is not used to provision real systems.
 For this [litmus](https://github.com/puppetlabs/puppet_litmus) is used.
 
-Litmus fires up fresh containers or virtual machines, applies this puppet
+Litmus fires up fresh containers or virtual machines, applies this Puppet
 module and then checks if it behaved as expected by running a series of tests
 on it. Those tests are implemented in `spec/acceptance/`.
```

Comment:
Since you converted `spec/unit` to a link, you could also convert this to a link.